liionpack/utils.py

```diff
@@ -99,6 +99,7 @@ def add_events_to_model(model):
     """
     for event in model.events:
         model.variables.update({"Event: " + event.name: event.expression})
+        model.event_names.append("Event: " + event.name)
```

Comment:
Presumably the issue arises because the events for the model are updated, but the event names are not updated. This may solve the issue if there is some way to append even names to the model.